endpoint.json

```diff
@@ -41,6 +61,12 @@
         "functions.js",
         "lex-helpers.js"
```

Comment:
Can we unify the lex-helpers.js helpers with the funtions.js?
This way we could use only the "_{method}" and remove the old "_http{Method}".

Reply:
Leave the previous helpers and do not remove the methods to support backwards compatibility.